GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -545,6 +581,14 @@ - (void)syncThreadUnsafeRemoveBatchWithID:(nonnull NSNumber *)batchID
       removeBatchDir(batchDirPath);
     }
   }
+
+  [self.sizeTracker resetCachedSize];
+}
+
+- (GDTCORStorageSizeBytes)fileSizeAtURL:(NSURL *)fileURL {
+  NSNumber *fileSize;
+  [fileURL getResourceValue:&fileSize forKey:NSURLFileSizeKey error:nil];
```

Comment:
Can we add this as a util method (may be even static) to the size tracker? Not too much sold on it. But to unify the size tracking/calculation logic to a single place.

Reply:
Done